src/lib/swankyCommand.ts

```diff
@@ -34,15 +36,36 @@ export abstract class SwankyCommand<T extends typeof Command> extends Command {
     this.args = args as Args<T>;
 
     this.logger = swankyLogger;
+    this.swankyConfig = {
+      node: {
+        polkadotPalletVersions: "",
+        localPath: "",
+        supportedInk: "",
+      },
+      contracts: {},
+      defaultAccount: null,
+      accounts: [],
+      networks: {},
+    };
+
+    try {
+      const localConfig = await getSwankyLocalConfig();
+      this.swankyConfig = {
+        ...this.swankyConfig,
+        ...localConfig,
+      };
+    } catch (error) {
+      this.logger.warn("No local config found")
+    }
+
     try {
-      this.swankyConfig = await getSwankyConfig();
+      const systemConfig = await getSwankySystemConfig();
+      this.swankyConfig = {
+        ...this.swankyConfig,
+        ...systemConfig,
+      };
     } catch (error) {
-      if (
-        error instanceof Error &&
-        error.message.includes("swanky.config.json") &&
-        (this.constructor as typeof SwankyCommand).ENSURE_SWANKY_CONFIG
```

Comment:
The ENSURE_SWANKY_CONFIG check must be kept, we can bypass it only for the commands that do not need swanky config files. You can use the same approach implemented in the init constructor.